### ISSUE TITLE
Some reading on enabling CORS

### ISSUE THREAD

**Issue by brody2consult** [apache/cordova-docs#1119]
I found this from a tweet: https://breautek.com/2020/07/14/enabling-cors/

I think this needs to be documented or linked to.

**Comment by breautek** [comment]
Agreed. We do have a blog post but it does need to be documented in the ios platform.

And I think we should probably have an example of the config for both nginx and Apache webserver, two which will probably a majority of server setups.

I'm not all that familiar with Apache webserver but I'm more than happy to pull parts of the my blog post for this.
